manifests/t/TuxGuitar/TuxGuitar/1.5.4/TuxGuitar.TuxGuitar.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TuxGuitar.TuxGuitar
+PackageVersion: 1.5.4
+PackageLocale: en-US
+Publisher: TuxGuitar
+PackageName: TuxGuitar
+License: LGPLv2
```

Comment:
```suggestion
License: LGPL-2.1-only
```

Reply:
Would be best to follow https://spdx.org/licenses/

I found `LGPL-2.1-only` from https://en.wikipedia.org/wiki/TuxGuitar

Reply:
I got LGPLv2 from their SourceForge, but indeed the official [website](http://www.tuxguitar.com.ar/license.html) specifies LGPLv2.1 or later.